src/components/common/BtnTask/BtnTask.tsx

```diff
@@ -41,6 +42,10 @@ function BtnTask(props: BtnTaskProps) {
 
 	/** 모달 띄우기 */
 	const handleDoubleClick = (e: React.MouseEvent) => {
+		if (onDoubleClick) {
+			onDoubleClick(e);
+			return;
+		}
```

Comment:
```suggestion
		if (preventDoubleClick) {
			e.preventDefault();
			return;
		}
```

위와 같이 수정하면 이 부분을 이렇게 고쳐볼 수 있을 것 같아요!

Reply:
또 event 중단 시 사용할 수 있는 방식이 여러가지 있는데, `stopPropagation()` 은 현재 이벤트가 상위로 전파되지 않도록 하는 것이고 `event.preventDefault()` 가 현재 이벤트의 기본 동작을 중단하는 방식이라 `preventDefault()` 를 사용하는 것은 어떨지 의견이 궁금해요!

Reply:
오,, 좋은 의견 감사합니다! 찾아보니 제안주신 `preventDefault()`는 고유의 동작을 막는 방식이라 요 로직에는 해당 방식이 더 적합할 것 같네요!! 반영하도록 하겠습니다.